modules/home-environment.nix

```diff
@@ -404,6 +419,31 @@ in
       }
     ];
 
+    warnings =
+      let
+        hmRelease = fileContents ../.release;
+        nixpkgsRelease = pkgs.lib.trivial.release;
```

Comment:
What's the difference with `pkgs.lib.version`?

Reply:
The `pkgs.lib.version` value is the same as `pkgs.lib.trivial.version`, for example, "21.05.git.e9148dc1c30" on my system.

The `pkgs.lib.trivial.release` value, on the other hand, is simply "21.05".

Reply:
IIRC the values are different with a nixpkgs flake. I'll have to check

Reply:
Ah, seems odd to me if those would differ in a flake context but it would certainly be good to verify, thanks!

Reply:
``` console
$ nix eval --impure --expr '(builtins.getFlake "nixpkgs").lib.version'
"21.11pre-git"

$ nix eval --impure --expr '(builtins.getFlake "nixpkgs").lib.trivial.release'
"21.11"
```

It only affects `lib.version` so it won't be a problem!